web/frontend/tests/tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
+  "compilerOptions": {
+    "target": "ES2022",
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "esnext"
+    ],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "strict": false,
```

Comment:
What happens if you set this to `true`? That's the interesting option!

Reply:
I just copy-pasted the config from the "main" project, I will test that

Reply:
changed it, not much change for the moment, though that might be because I ran the linter and fixed everything beforehand